### DISCUSSION TITLE
GITDIS google doc

### DISCUSSION THREAD

**Discussion by qwertychomp** [Command-Enterprises/Commander#13] [Help]
instead of directing the user to a google doc with a link to this discussions thing, can i just replace it with a script that just redirects? it would make much more sense

**Comment by accessgames** [comment]
Yea sure that makes sense. Im also gonna update the games page in the morning to actually work if at @Quartinal   doesn't respond to this discussion because I dont think he is working on it

**Comment by accessgames** [comment]
wait are you the same account as @Quartinal ? if so plz tell me but if not then im just going to get off for the night its late. I'll keep is scroll affect but also have the games there as well

**Reply by qwertychomp** [reply]
no, im sequoia (froms daves bus :) )

**Reply by accessgames** [reply]
oh ok

**Comment by qwertychomp** [comment]
implemented in my latest pull request, closing discussion
